randovania/games/dread/logic_database/Cataris.txt

```diff
@@ -2815,11 +2815,11 @@ Extra - asset_id: collision_camera_040
       Can Slide
   > Event - Push Wide Beam Block
       All of the following:
-          Wave Beam and Can Slide
+          Wave Beam
           Any of the following:
               Push Wide Beam Block
```

Comment:
one last thing, this can be changed to just charge beam since you dont need wide for it.

Reply:
leave the wide req. there's no telling whether later patcher changes will introduce the requirement again and it shouldn't make too much difference right now

Reply:
I think since the Beam Split bug isn't intentional and there is effort to fix it, the logic should be ignorant of the bug.

Reply:
For reference, I think this would have been added as viable with just charge and wave beam, if the data visualizer map had shown the wide beam block in its correct location. At the time the split beams were added, we thought that pushing wide beam blocks with wave beam was ok from a sharp angle, so in a bunch of sitations, that was added to logic, just not here. The map looks like this 
![image](https://github.com/randovania/randovania/assets/6470224/bb87aa0e-0bb7-425d-bc5f-8a378b1256b9) while it looks like this ingame 
![image](https://github.com/randovania/randovania/assets/6470224/2af27510-7bb2-4191-8fff-24bdc5555b42)
However, with what we know now, I think it's for the better to just leave it as is and wait for a fix for the wideless shenanigans, as it's probably overall better for the item balance if it requires wide beam.